addons/mail/static/src/models/message_tracking_value/message_tracking_value_tests.js

```diff
@@ -0,0 +1,517 @@
+/** @odoo-module **/
+import { insert } from '@mail/model/model_field_command';
+import {
+    afterEach,
+    beforeEach,
+    nextAnimationFrame,
+    start,
+} from '@mail/utils/test_utils';
+
+QUnit.module('mail', {}, function () {
+QUnit.module('models', {}, function () {
+QUnit.module('message_tracking_value', {}, function () {
+QUnit.module('message_tracking_value_tests.js', {
+    beforeEach() {
+        beforeEach(this);
+
+        this.start = async params => {
+            const { env, widget } = await start(Object.assign({}, params, {
+                data: this.data,
+            }));
+            this.env = env;
+            this.widget = widget;
+        };
+    },
+    afterEach() {
+        afterEach(this);
+    },
+});
+
+QUnit.test('model test of tracking value (float type)', async function (assert) {
+    assert.expect(6);
+
+    await this.start();
+    const message = this.env.models['mail.message'].create({
+        id: 11,
+        trackingValues: insert({
+            changedField: "Total",
+            fieldType: "float",
+            id: 6,
+            newValue: 45.67,
+            oldValue: 12.3,
+        }),
+    });
+    assert.ok(this.env.models['mail.message_tracking_value'].findFromIdentifyingData({ id: 6 }));
+    assert.ok(message);
```

Comment:
Also useless assert, the goal is not to test that `create()` works.
Apply same comment in all tests.